save-backend/src/test/kotlin/com/saveourtool/save/backend/utils/SecurityTestingUtils.kt

```diff
@@ -4,14 +4,19 @@
 
 package com.saveourtool.save.backend.utils
 
+import com.saveourtool.save.authservice.utils.AuthenticationUserDetails
 import org.springframework.security.authentication.UsernamePasswordAuthenticationToken
 import org.springframework.security.core.context.SecurityContextHolder
 
 /**
  * @param action mutation for a mocked user in test security context
  */
-internal fun mutateMockedUser(action: UsernamePasswordAuthenticationToken.() -> Unit) {
+internal fun mutateMockedUser(id: Long, name: String = "N/A") {
     SecurityContextHolder.getContext().apply {
-        authentication = (authentication as UsernamePasswordAuthenticationToken).apply(action)
+        authentication = AuthenticationUserDetails(
```

Comment:
currently we do:
```kotlin
@WithMockUser(name = "something", roles = ["SINGLE_ROLE"])
fun test() {
    mutateMockedUser(id = 99)
    TODO()
}
```

It can be refactored to:
```kotlin
fun test() {
    mutateMockedUser(id = 99, name = "something", roles = "SINGLE_ROLE")
    TODO()
}
```